drivers/interrupt_controller/intc_exti_stm32.c

```diff
@@ -412,17 +415,18 @@ static int stm32_exti_init(const struct device *dev)
 }
 
 static struct stm32_exti_data exti_data;
-DEVICE_DEFINE(exti_stm32, STM32_EXTI_NAME, stm32_exti_init,
-	      NULL, &exti_data, NULL,
-	      PRE_KERNEL_1, CONFIG_KERNEL_INIT_PRIORITY_DEVICE,
-	      NULL);
+DEVICE_DT_DEFINE(DT_NODELABEL(exti), &stm32_exti_init,
```

Comment:
Is there a reason to use DT_NODELABEL here?  Can you not use the `DEVICE_DT_INST_DEFINE` variant since you set `DT_DRV_COMPAT`?

Reply:
Well, I though `_INST_` was meaningless here as there is only one instance.

Reply:
If you don't want to use the `_INST_`.  I'd maybe suggest doing something like:

`#define EXTI_NODE DT_INST(0)`

and using `EXTI_NODE` elsewhere so at least you are finding the node based on the compat value.

Reply:
ok, this is a good alternative